config/linker.in

```diff
@@ -0,0 +1,3 @@
+menu "Linkers"
```

Comment:
We probably want some indication that there are other linkers. Ideally we'd set things up so bfd, gold and mold are all selectable here. But as the former two are part of binutils it's probably not super easy to do. As a compromise "External Linkers", "Extra Linkers" or "Additional Linkers" might be a better name. Or maybe `comment "BFD enabled in binutils" if BINUTILS_LINKER_LD` and `comment "GOLD enabled in binutils" if BINUTILS_LINKER_GOLD` and

Reply:
I can add these, no problem.

Reply:
Adjusted to handle enabling both binutils linkers as well.

And syntax that was accepted on my workstation.